src/DynamoCore/Interfaces/ICustomNodeManager.cs

```diff
@@ -8,6 +8,6 @@ public interface ICustomNodeManager
     {
         IEnumerable<CustomNodeInfo> AddUninitializedCustomNodesInPath(string path, bool isTestMode, bool isPackageMember = false);
         Guid GuidFromPath(string path);
-        bool TryGetFunctionWorkspace(Guid id, bool isTestMode, out ICustomNodeWorkspaceModel ws);
+        bool TryGetFunctionWorkspace(Guid id, bool isTestMode, out CustomNodeWorkspaceModel ws);
```

Comment:
Let's not reverse this work. This should be `ICustomNodeWorkspaceModel`, although this requires a cast in `DynamoPublish`.